src/Upload/views.py

```diff
@@ -20,7 +20,14 @@ def upload_file(request):
 
 
 def upload_view(request):
-    Uploads = Upload.objects.all()
+
+    Uploads = Upload.objects.filter(name = request.user)
```

Comment:
I see that the upload can be found based on the user. That makes my above comment not valid. Very nice use of a foreign key.